src/test/java/uk/gov/justice/hmpps/prison/service/ReleaseAPrisonerIntegrationTest.java

```diff
@@ -146,6 +163,28 @@ public void deactivateAllProgramProfiles() {
         assertThat(activeProgramProfiles).isZero();
     }
 
+    @Test
+    public void dontDeactivateAllProgramProfiles() {
```

Comment:
Test copied from an existing test but turns on the feature switch for DPS Activities first.